src/models/feature.h

```diff
@@ -34,11 +34,8 @@ class Feature : public QObject {
   Q_PROPERTY(QString releaseVersion READ releaseVersion CONSTANT)
   Q_PROPERTY(bool isReleased MEMBER m_released CONSTANT)
   Q_PROPERTY(bool isNew READ isNew CONSTANT)
-  Q_PROPERTY(bool flippableOn READ isFlippableOn CONSTANT)
-  Q_PROPERTY(bool flippableOff READ isFlippableOff CONSTANT)
+  Q_PROPERTY(bool isToggleable READ isToggleable CONSTANT)
```

Comment:
If we have `isToggleable`, do we still need `isFlippableOn` and `isFlippableOff` ?